Automattic/jetpack
# Boost: Add Premium CTA section to Boost Critical CSS module

Fixes #23317 

#### Changes proposed in this Pull Request:

* Add a call to action panel to the Critical CSS module, as per the Figma designs: p1HpG7-f1L-p2
* Creates the CTA section and the appropriate styles. This ticket does not handle the rest of the flow.

#### Jetpack product discussion


pc9hqz-iD-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Make sure that Cloud CSS is disabled. Set `JETPACK_BOOST_CLOUD_CRITICAL_CSS` as `false`.
* Visit Jetpack Boost dashboard page.
* Ensure the CTA section appears below the Critical CSS module as per p1HpG7-f1L-p2
* Enable Cloud CSS. Set `JETPACK_BOOST_CLOUD_CRITICAL_CSS` as `true`.
* Ensure the CTA section does not appear.
